### PR TITLE
[IMP] website_sale: reactivate cart testing tour

### PR DESCRIPTION
**Pull request by Demesmaeker**
task-3547913

### CODE REVIEW

File: addons/website_sale/static/tests/tours/website_sale_add_to_cart_snippet_tour.js [hunk 1]
```diff
@@ -44,6 +44,11 @@ wTourUtils.registerWebsitePreviewTour('add_to_cart_snippet_tour', {
         ...wTourUtils.selectElementInWeSelectWidget('action_picker_opt', 'Buy Now'),
         ...wTourUtils.clickOnSave(),
         wTourUtils.clickOnElement('add to cart button', 'iframe .s_add_to_cart_btn'),
+        // TODO edm: make it wait for the page to load
```
Comment (vava-odoo) [inline addons/website_sale/static/tests/tours/website_sale_add_to_cart_snippet_tour.js:47]: :eyes:

File: addons/website_sale/static/tests/tours/website_sale_add_to_cart_snippet_tour.js [hunk 2]
```diff
@@ -44,6 +44,11 @@ wTourUtils.registerWebsitePreviewTour('add_to_cart_snippet_tour', {
...
+        {
+            trigger: "body",
+            isCheck: true,  // wait for the page to load, as the next check was sometime too fast
```
Comment (vava-odoo) [inline addons/website_sale/static/tests/tours/website_sale_add_to_cart_snippet_tour.js:50]:
```suggestion
            isCheck: true,  // wait for the page to load, as the next check was sometimes too fast
```

File: addons/website_sale/static/tests/tours/website_sale_add_to_cart_snippet_tour.js (hunk 1 above)
Comment (Demesmaeker) [reply addons/website_sale/static/tests/tours/website_sale_add_to_cart_snippet_tour.js:47]: :facepalm:

**Review by vava-odoo — APPROVED** [review]
Looks correct, and multi build agrees.

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Comment by vava-odoo** [comment]
I think it should be retargeted to 17.0.

**Comment by Demesmaeker** [comment]
Indeed, I did go a bit too fast, will first run the multi-build multiple times before rebasing, ty ^^

**Comment by AntoineVDV** [comment]
@robodoo delegate+

File: addons/website_sale/static/tests/tours/website_sale_add_to_cart_snippet_tour.js [hunk 3]
```diff
@@ -32,23 +32,29 @@ wTourUtils.registerWebsitePreviewTour('add_to_cart_snippet_tour', {
...
-        ...wTourUtils.selectElementInWeSelectWidget('product_template_picker_opt', 'Conference Chair', true),
-        ...wTourUtils.selectElementInWeSelectWidget('product_variant_picker_opt', 'Conference Chair (Aluminium)'),
-        ...wTourUtils.clickOnSave(),
-        wTourUtils.clickOnElement('add to cart button', 'iframe .s_add_to_cart_btn'),
+        // ...editAddToCartSnippet(),
+        // ...wTourUtils.selectElementInWeSelectWidget('product_template_picker_opt', 'Conference Chair', true),
+        // ...wTourUtils.selectElementInWeSelectWidget('product_variant_picker_opt', 'Conference Chair (Aluminium)'),
+        // ...wTourUtils.clickOnSave(),
+        // wTourUtils.clickOnElement('add to cart button', 'iframe .s_add_to_cart_btn'),
+        // TODO edm: re-enable this part when this isn't an indeterminist error anymore
```
Comment (vava-odoo) [inline addons/website_sale/static/tests/tours/website_sale_add_to_cart_snippet_tour.js:40]: :eyes:

**Comment by Demesmaeker** [comment]
@robodoo r+

**Pull request closed without merging**